app/code/Magento/WishlistGraphQl/etc/schema.graphqls

```diff
@@ -5,6 +5,19 @@ type Query {
     wishlist: WishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\WishlistResolver") @doc(description: "The wishlist query returns the contents of a customer's wish list") @cache(cacheable: false)
 }
 
+type Mutation {
+    addItemsToWishlist (input: AddItemsToWishlistInput): AddItemToWishlistOutput @resolver(class: "\\Magento\\WishlistGraphQl\\Model\\Resolver\\AddItemsToWishlist") @doc(description:"Add items wishlist")
+}
+
+input AddItemsToWishlistInput {
+    wishlist_items: [WishlistItemInput!]!
```

Comment:
Field arguments support extension so there is no need to create `AddItemsToWishlistInput` wrapper, please use `wishlist_items` as an argument directly.